2024-11-17 18:44:23,831 INFO    MainThread:41808 [wandb_setup.py:_flush():79] Current SDK version is 0.18.7
2024-11-17 18:44:23,831 INFO    MainThread:41808 [wandb_setup.py:_flush():79] Configure stats pid to 41808
2024-11-17 18:44:23,832 INFO    MainThread:41808 [wandb_setup.py:_flush():79] Loading settings from C:\Users\dnlal\.config\wandb\settings
2024-11-17 18:44:23,832 INFO    MainThread:41808 [wandb_setup.py:_flush():79] Loading settings from c:\Users\dnlal\OneDrive\Documents\GitHub\Proyecto3-IA\notebooks\wandb\settings
2024-11-17 18:44:23,832 INFO    MainThread:41808 [wandb_setup.py:_flush():79] Loading settings from environment variables: {}
2024-11-17 18:44:23,832 INFO    MainThread:41808 [wandb_setup.py:_flush():79] Applying setup settings: {'mode': None, '_disable_service': None}
2024-11-17 18:44:23,832 INFO    MainThread:41808 [wandb_setup.py:_flush():79] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-11-17 18:44:23,832 INFO    MainThread:41808 [wandb_setup.py:_flush():79] Applying login settings: {}
2024-11-17 18:44:23,832 INFO    MainThread:41808 [wandb_init.py:_log_setup():533] Logging user logs to c:\Users\dnlal\OneDrive\Documents\GitHub\Proyecto3-IA\notebooks\wandb\run-20241117_184423-59tn7503\logs\debug.log
2024-11-17 18:44:23,832 INFO    MainThread:41808 [wandb_init.py:_log_setup():534] Logging internal logs to c:\Users\dnlal\OneDrive\Documents\GitHub\Proyecto3-IA\notebooks\wandb\run-20241117_184423-59tn7503\logs\debug-internal.log
2024-11-17 18:44:23,832 INFO    MainThread:41808 [wandb_init.py:init():619] calling init triggers
2024-11-17 18:44:23,832 INFO    MainThread:41808 [wandb_init.py:init():626] wandb.init called with sweep_config: {}
config: {'defaults': ['_self_', {'model': 'unet'}, {'training': 'default'}], 'data': {'data_dir': 'Butterfly-dataset', 'batch_size': 32, 'num_workers': 4, 'image_size': 224, 'unlabeled_ratio': 0.7}, 'wandb': {'project': 'butterfly-classification', 'entity': None, 'mode': 'online'}, 'seed': 42, 'autoencoder': {'in_channels': 3, 'latent_dim': 256, 'hidden_dims': [64, 128, 256, 512]}, 'classifier': {'num_classes': 20, 'dropout': 0.5}, 'training': {'max_epochs': 50, 'early_stopping_patience': 5, 'learning_rate': 0.001, 'weight_decay': 0.0001, 'gradient_clip_val': 1.0}}
2024-11-17 18:44:23,832 INFO    MainThread:41808 [wandb_init.py:init():669] starting backend
2024-11-17 18:44:23,832 INFO    MainThread:41808 [wandb_init.py:init():673] sending inform_init request
2024-11-17 18:44:23,836 INFO    MainThread:41808 [wandb_init.py:init():686] backend started and connected
2024-11-17 18:44:23,842 INFO    MainThread:41808 [wandb_run.py:_label_probe_notebook():1339] probe notebook
2024-11-17 18:44:23,842 INFO    MainThread:41808 [wandb_run.py:_label_probe_notebook():1349] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2024-11-17 18:44:23,842 INFO    MainThread:41808 [wandb_init.py:init():781] updated telemetry
2024-11-17 18:44:23,888 INFO    MainThread:41808 [wandb_init.py:init():814] communicating run to backend with 900.0 second timeout
2024-11-17 18:59:24,643 ERROR   MainThread:41808 [wandb_init.py:init():841] encountered error: Run initialization has timed out after 900.0 sec. Please try increasing the timeout with the `init_timeout` setting: `wandb.init(settings=wandb.Settings(init_timeout=120))`.
2024-11-17 18:59:24,643 ERROR   MainThread:41808 [wandb_init.py:init():1266] error in wandb.init()
Traceback (most recent call last):
  File "c:\Users\dnlal\AppData\Local\Programs\Python\Python312\Lib\site-packages\wandb\sdk\wandb_init.py", line 1256, in init
    return wi.init()
           ^^^^^^^^^
  File "c:\Users\dnlal\AppData\Local\Programs\Python\Python312\Lib\site-packages\wandb\sdk\wandb_init.py", line 847, in init
    raise error
wandb.errors.errors.CommError: Run initialization has timed out after 900.0 sec. Please try increasing the timeout with the `init_timeout` setting: `wandb.init(settings=wandb.Settings(init_timeout=120))`.
